### PR TITLE
Made trashcan image black by adding new image and changing opacity.

### PR DESCRIPTION
**Pull request by ramiAbdou**
This PR:
1) Replaced the sprites.png image with a blacked out version.
2) Changed the opacity of the trashcan.
3) Added a blocklyZoomButton class to all of the zoom buttons to allow for the cursor to change to pointer on hover in CSS file called 'ai2blockly.css' in the blocklyeditor->media folder.

### CODE REVIEW

**Comment by ramiAbdou** [comment]
@ewpatton @josmas

**Comment by josmas** [comment]
Hey @ramiAbdou as in the other PR, there are a lot of changes in here that are just formatting. Could you please review and commit only actual changes? Thanks!

**Comment by josmas** [comment]
There also seems to be something weird with your local state, as this PR has 66 commits instead of just your own.

**Comment by ramiAbdou** [comment]
@josmas hey Jose, any idea on how to undo the other 66 commits? not exactly sure how they got into my PR.

**Comment by josmas** [comment]
Not really sure what happened there. You can try and _cd lib/blockly_, then _checkout mastermit_, then create a new branch with _git checkout -b sidebarv2_ and then try to rebase your sidebar branch into sidebarv2. Rebase should grab your latest commits and push them to the top of the new branch, but you may have to experiment a little! :)

**Comment by ramiAbdou** [comment]
Hmm okay, I'll try those things. Is there any way to delete this pull request and just make another pull request without all those other 66 commits?

**Comment by ewpatton** [comment]
You should be able to update the base branch of the pull request to be mitmaster instead of master.

**Comment by ramiAbdou** [comment]
@ewpatton This solved the problem thanks!

File: core/trashcan.js [hunk 1]
```diff
@@ -166,8 +166,9 @@ Blockly.Trashcan.prototype.createDom = function() {
   */
   this.svgGroup_ = Blockly.utils.createSvgElement('g',
       {'class': 'blocklyTrash'}, null);
+  var clip;
```
Comment (ewpatton) [inline core/trashcan.js]: While it's really tempting to do this (I have the same impulse), we try to not introduce unnecessary changes to this repository to make it easier to merge with Blockly. Please return the repeated var declaration.

File: core/trashcan.js [hunk 2]
```diff
@@ -329,4 +330,4 @@ Blockly.Trashcan.prototype.click = function() {
     return;
   }
   console.log('TODO: Inspect trash.');
-};
+};
```
Comment (ewpatton) [inline core/trashcan.js]: The newline at the end of this file was removed.

File: core/zoom_controls.js [hunk 3]
```diff
@@ -115,9 +115,10 @@ Blockly.ZoomControls.prototype.createDom = function() {
   */
   this.svgGroup_ = Blockly.utils.createSvgElement('g',
       {'class': 'blocklyZoom'}, null);
+  var clip;
```
Comment (ewpatton) [inline core/zoom_controls.js]: See previous comment about var.

File: core/zoom_controls.js [hunk 4]
```diff
@@ -243,4 +247,4 @@ Blockly.ZoomControls.prototype.position = function() {
   }
   this.svgGroup_.setAttribute('transform',
       'translate(' + this.left_ + ',' + this.top_ + ')');
-};
+};
```
Comment (ewpatton) [inline core/zoom_controls.js]: Newline.

**Review by ewpatton — CHANGES_REQUESTED** [review]
A few minor code style issues but otherwise LGTM.

File: core/css.js [hunk 5]
```diff
@@ -957,4 +957,4 @@ Blockly.Css.CONTENT = [
   '}',
 
   ''
-];
```
Comment (ewpatton) [inline core/css.js]: @ramiAbdou This line is still showing as having the final line ending removed. Can you check your code editor's settings to make sure that it will retain the trailing newline? It might be removing it based on a configuration option.

Comment (ramiAbdou) [reply core/css.js]: Should be solved now.

**Review by ewpatton — APPROVED** [review]
LGTM
